src/sys/uio.rs

```diff
@@ -19,7 +19,16 @@ pub fn readv(fd: RawFd, iov: &mut [IoVec<&mut [u8]>]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
-#[cfg(target_os = "linux")]
```

Comment:
Doc comments should start with a short summary line, like `git` commit messages. Then a blank line and a longer descriptive message afterwards. See [the Rust book](https://doc.rust-lang.org/book/first-edition/documentation.html#writing-documentation-comments).